CONTRIBUTING.md

```diff
@@ -78,12 +78,21 @@ Running the tests
 
 To automatically find all files named `_test.dart` inside a package and run them inside the flutter shell as a test use the `flutter test` command, e.g:
 
- * `cd packages/newton`
+ * `cd examples/stocks`
  * `flutter test`
```

Comment:
Does this do anything useful today?  If not, we should make it do something useful soon.